---
bool success
point_cloud_proc/Plane plane_object
